### ISSUE TITLE
[ic-top-navigation]: Focus styling is cut off on mobile for app logo/app title. 

### ISSUE THREAD

**Issue by jd239** [mi6/ic-ui-kit#540]
## Summary of the bug
It is an accessibility violation if a focus state is cut off. Clicking the app title on `ic-top-navigation` on mobile cuts off the left hand side of the focus state. 

## 🪜 How to reproduce
Tell us the steps to reproduce the problem:
1. Go to page: https://design.sis.gov.uk/
2. Click on the app title in the top left

## 🧐 Expected behaviour
The focus state on the app logo and app title should be on display without being cut off. 

## 📸 Screenshots or code
![Capture](https://user-images.githubusercontent.com/116016939/229147393-a2ea5b6b-02b7-4c75-a7d3-0bd15bdede67.PNG)

## 🖥 Desktop
 - OS: N/A
 - Browser: N/A
 - Version: N/A

## 📱 Mobile
 - Device: Android
 - OS: Android
 - Browser: Chrome
 - Version: 111

## Additional info
The viewport to see the issue is 393 x 851.

**Comment by ASM995** [comment]
Work complete.
